ChronoKeeper/KeeperRegClient.cpp

```diff
@@ -0,0 +1,7 @@
+
+int main(int argc, char** argv) {
+    if(argc != 3) {
+        std::cerr << "Usage: " << argv[0] << " <address> <provider_id>" << std::endl;
+        exit(0);
+    }
+    tl::engine myEngine("ofi+sockets", THALLIUM_CLIENT_MODE);
```

Comment:
I know this file is not included in any build target. But at least close this main function with a matching brace.